brjs-core-tests/src/test-integration/java/org/bladerunnerjs/memoization/WatchingFileModificationObserverThreadTest.java

```diff
@@ -297,7 +297,7 @@ private boolean waitForUpdate(File expectedFile, boolean mustBeIndex0) throws In
 			checkForUpdates(1);
 			boolean foundFileChange = (mustBeIndex0) ? fileChanges.indexOf(expectedFile) == 0 : fileChanges.contains(expectedFile);
 			if (fileChanges.size() > 0 && foundFileChange) {
-				fileChanges.remove(foundFileChange);
+				fileChanges.remove(expectedFile);
```

Comment:
Do we know why this didn't break anything? Do we even need `fileChanges.remove(...)`?